components/kitchen-sink.tsx

```diff
@@ -1234,14 +1259,25 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                             value={errorEmailInputText}
                             onChangeText={(text: string): void => setErrorEmailInputText(text)}
                             error
+                            allowFontScaling={!DISABLE_FONT_SCALE}
+                            maxFontSizeMultiplier={MAX_FONT_SCALE}
+                        />
+                        <TextInput
+                            label="Name Disabled"
+                            value="John Smith"
+                            style={{ marginTop: 24 }}
+                            disabled
+                            allowFontScaling={!DISABLE_FONT_SCALE}
+                            maxFontSizeMultiplier={MAX_FONT_SCALE}
                         />
-                        <TextInput label="Name Disabled" value="John Smith" style={{ marginTop: 24 }} disabled />
```

Comment:
What was the motivation for removing this input?

Reply:
There is no removal, adding two more props formatted all the props to move on separate lines.
Here's attached screenshot of changes:
<img width="1329" alt="Screenshot 2022-12-23 at 7 17 17 PM" src="https://user-images.githubusercontent.com/25982779/209346553-327dc586-ee66-41be-bdb9-963d44c607a4.png">